File: C/linearSearch.c
#include <stdio.h>
 
#define N 100

int search(int arr[], int n, int x)
{
    int i;
    for (i = 0; i < n; i++)
        if (arr[i] == x)
            return i;
    return -1;
}
 
// Driver code
int main(void)
{
    int arr[N];
    printf("Enter size of array: ");
    int n;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &arr[i]);
    }

    printf("Enter element to be searched: ");
    int x;
    scanf("%d", &x);
    n = sizeof(arr) / sizeof(arr[0]);
   
    // Function call
    int result = search(arr, n, x);
    (result == -1)
        ? printf("Element is not present in array\n")
        : printf("Element is present at index %d\n", result);
    return 0;
}

File: Python/linearSearch.py
def linear_search(values, search_for):
    search_at = 0
    search_res = False

# Match the value with each data element	
    while search_at < len(values) and search_res is False:
        if values[search_at] == search_for:
            print("Element is present at index", str(search_at))
            search_res = True
        else:
            search_at = search_at + 1

    return search_res

arr = []
n = int(input("Enter number of elements: "))
print("Enter elements in sorted order\n")
# iterating till the range 
for i in range(0, n): 
    ele = int(input()) 
    arr.append(ele) # adding the element 
x = int(input("Enter key to search: "))

if(not(linear_search(arr, x))):
    print("Element is not present in array")